backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -50,14 +53,42 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.MarketedOn, src => src.MarketedOn)
                 .Map(dest => dest.OfferAcceptedOn, src => src.OfferAcceptedOn)
                 .Map(dest => dest.DisposedOn, src => src.DisposedOn)
+                .Map(dest => dest.AssessedOn, src => src.AssessedOn)
+                .Map(dest => dest.AdjustedOn, src => src.AdjustedOn)
+                .Map(dest => dest.PreliminaryFormSignedOn, src => src.PreliminaryFormSignedOn)
+                .Map(dest => dest.FinalFormSignedOn, src => src.FinalFormSignedOn)
+                .Map(dest => dest.PriorYearAdjustmentOn, src => src.PriorYearAdjustmentOn)
                 .Map(dest => dest.ExemptionRequested, src => src.ExemptionRequested)
                 .Map(dest => dest.ExemptionRationale, src => src.ExemptionRationale)
                 .Map(dest => dest.NetBook, src => src.NetBook)
+                .Map(dest => dest.NetBookOverride, src => src.NetBookOverride)
```

Comment:
What is the NetBookOverride?

Reply:
This is documented in the models, but if netbookoverride the api will use a static value for the netbook instead of a calculated value. Some sort of boolean flag is required here because the frontend needs a persistent way to communicate to the backend that sums should no longer be used.

Reply:
I don't understand the need still. If I remember correctly, SRES is just going to put a number in here and that's the official `NetBook` that will be used for all reporting.  I think for now the backend will just have to not sum the `NetBook` value due to this requirement.  I think it can still sum the other values.

Reply:
If this is what is desired, then I will also have to remove/rename the "Sum Netbook" column from the approval requests and project list tables as well. Not sure if it makes sense to continue to include that column at all if SRES is the only one entering a value.

Reply:
Would this apply for assessedOverride as well? that boolean is being used for essentially the same purpose, just for assessed instead of netbook. I'm actually not sure if the summed value for that is being used in the frontend.

Reply:
We don't have a place to enter the value at that point.  The value is coming from the owning agency updating their properties.  For some reason when they sell they are only receiving a final number.  I think it's an issue with their process, but for now the only single `NetBook` value we care about at the point of SPL disposal is the one they manually enter once.

Reply:
Before I make this change, please confirm the following:
1) I should remove all evaluation/fiscal summation logic from the backend. (netbook, assessed, estimated)
2) I should remove all frontend logic that depends on # 1, such as the project list view.

Reply:
Based on our last conversation with Mikaela, don't remove any summation logic.  Add a new separate `Project.CloseOutNetBook` property to capture the value they enter on the Close Out Form.